src/libcore/num/mod.rs

```diff
@@ -930,6 +930,62 @@ $EndFeature, "
             }
         }
 
+        doc_comment! {
+            concat!("Saturating integer negation. Computes `-self`, returning `MAX` if `self == MIN`
+instead of overflowing.
+
+# Examples
+
+Basic usage:
+
+```
+", $Feature, "#![feature(saturating_neg)]
+assert_eq!(100", stringify!($SelfT), ".saturating_neg(), -100);
+assert_eq!((-100", stringify!($SelfT), ").saturating_neg(), 100);
+assert_eq!(", stringify!($SelfT), "::min_value().saturating_neg(), ", stringify!($SelfT),
+"::max_value());
+assert_eq!(", stringify!($SelfT), "::max_value().saturating_neg(), ", stringify!($SelfT),
+"::min_value() + 1);",
+$EndFeature, "
+```"),
+
+            #[unstable(feature = "saturating_neg", issue = "0")]
```

Comment:
Do we want to make #59983 the tracking issue or is a seperate issue desired?

Reply:
My idea was to put the issue ID of this PR therein. If using #59983 is preferred I will happily use that.

Reply:
This was more a question to the reviewer. I'm not a core member and therefore cannot decide whether to use `0` for the moment or the Issue. But a *trackig issue* is always an **issue** number, never a PR.